public/index.php

```diff
@@ -16,21 +22,15 @@
     $_SERVER['SCRIPT_NAME'] = '/index.php';
 }
 
-require_once dirname(__DIR__) . '/vendor/autoload.php';
-
-/**
- *  Set debug value for web application runner, for default its `true` add additionally the validation of the
- *  container-di configurations (debug mode).
- */
-define('YII_DEBUG', getenv('YII_DEBUG') ?: true);
+require_once dirname(__DIR__) . '/bootstrap.php';
 
-/**
- *  Set environment value for web application runner, for default its `null`.
- *
- *  @link https://github.com/yiisoft/config#environments
- */
-define('YII_ENV', getenv('YII_ENV') ?: null);
+if ($_ENV['YII_ENV'] === 'test') {
```

Comment:
For use `YII_ENV=test` need create environment `test` for config plugin. Otherwise, there will be an error "environment does not exist".

Reply:
Yes, you are right. But this is a different task, see https://github.com/yiisoft/app/issues/166.